modules/living_spaces_activity/js/living_spaces_activity_load.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file
+ * Contains living_spaces_activity_load.js.
+ */
+
+(function ($, Drupal, drupalSettings) {
+
+  /**
+   * Loads PDM messages via ajax.
+   */
+  Drupal.behaviors.livingSpacesActivityLoad = {
+    attach: function (context, settings) {
+      setInterval(function() {
+        $.ajax({
+          url: Drupal.url('living-spaces-activity/get-persistent-messages/' + settings.user.uid),
+          type: 'POST',
+          contentType: 'application/json; charset=utf-8',
+          dataType: 'json',
+          success: function success(value) {
+            if (value.success) {
+              if ($('div[id^=block-views-block-message-persistent]').length) {
+                $('div[id^=block-views-block-message-persistent]').replaceWith(value.message);
+              }
+              else {
+                $('div.highlighted').append(value.message);
+              }
+
+              Drupal.attachBehaviors();
+            }
+          }
+        });
+      }, 60 * 1000);
+    }
+  };
```

Comment:
We should not use set interval here. We should be using nodejs for this as we used it to update notifications bell and dropdown in Drupal.Nodejs.callbacks.livingSpacesActivity